### PR TITLE
Created RPC endpoint for current gas price

### PR DESCRIPTION
**Pull request by kmazurek**
Fixes: #3624

### CODE REVIEW

**Comment by mdtanrikulu** [comment]
But why we have another RPC for it? I was expecting to have the parameters in [this RPC](https://github.com/golemfactory/golem/issues/3624#issuecomment-443711243).

**Comment by kmazurek** [comment]
@mdtanrikulu I created a separate endpoint since the data returned by `comp.tasks.estimated.cost` is specific to tasks, while the added `pay.gas_price` is general and could be used independently.

File: tests/golem/test_client.py [hunk 1]
```diff
@@ -131,6 +131,18 @@ class TestClient(TestClientBase):
...
+        test_gas_price = 1234
+        test_price_limit = 12345
+        ets = self.client.transaction_system
+        ets.gas_price = test_gas_price
+        ets.gas_price_limit = test_price_limit
+
+        result = self.client.get_gas_price()
+
+        assert result["current_gas_price"] == str(test_gas_price)
+        assert result["gas_price_limit"] == str(test_price_limit)
```
Comment (jiivan) [inline tests/golem/test_client.py]: I generally prefer `self.assert*` methods from `unittest.TestCase`.

**Review by jiivan — APPROVED** [review]

File: tests/golem/test_client.py [hunk 2]
```diff
@@ -140,8 +140,8 @@ def test_get_gas_price(self, *_):
 
         result = self.client.get_gas_price()
 
-        assert result["current_gas_price"] == str(test_gas_price)
-        assert result["gas_price_limit"] == str(test_price_limit)
+        self.assertEquals(result["current_gas_price"], str(test_gas_price))
```
Comment (jiivan) [inline tests/golem/test_client.py]: `assertEquals` is deprecated in favor of `assertEqual` (without s)

Comment (kmazurek) [reply tests/golem/test_client.py]: Good catch, sorry

**Pull request merged**